templates/CONTRIBUTING.md

```diff
@@ -1,3 +1,57 @@
+# Contributing
+Welcome and thank you for your interest in contributing to TeamsFx templates! Before contributing to this project, please review this document for policies and procedures which will ease the contribution and review process for everyone. If you have questions, please raise your issue on github.
+
+## Built the project
+
+1. Clone this repo locally. (git clone https://github.com/OfficeDev/TeamsFx.git)
+1. Open a terminal and move into your local copy. (cd TeamsFx)
```

Comment:
The numbers are all 1.

Reply:
It's ok and has better maintainability. Markdown will automatically order it.